chain-signatures/node/Cargo.toml

```diff
@@ -28,6 +28,10 @@ semver = "1.0.23"
 sysinfo = "0.32.0"
 tokio-retry = "0.3"
 tracing-stackdriver = "0.10.0"
+alloy = "0.9.1"
+alloy-sol-types = "0.8.24"
+helios = { git = "https://github.com/a16z/helios", branch = "master" }
```

Comment:
we should fork this repo into our org, especially since we're targeting master branch here which will very likely change from underneath us and we'd be beholden to all those breaking changes